app/routes.js

```diff
@@ -24,21 +28,14 @@ module.exports = function(app) {
         res.sendfile('index.html');
     });
 
-    // get all screen ids
-    app.get('/api/screen_ids', function(req, res) {
+    // get fields of all screens
+    app.get('/api/screens/:field/', function(req, res) {
         Screen
             .find({})
-            .select('_id')
+            .select(req.params.field)
```

Comment:
definitely needs clarification — can only think of microscope tiles, which I don't think is what you're talking about here!